arrow/src/test/java/org/apache/iceberg/arrow/vectorized/VectorHolderTest.java

```diff
@@ -0,0 +1,127 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.iceberg.arrow.vectorized;
+
+import static org.mockito.Mockito.when;
+
+import org.apache.arrow.vector.FieldVector;
+import org.apache.iceberg.types.Type;
+import org.apache.iceberg.types.Types;
+import org.apache.parquet.column.ColumnDescriptor;
+import org.apache.parquet.column.Dictionary;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+class VectorHolderTest {
+  @Mock ColumnDescriptor columnDescriptor;
+  @Mock FieldVector vector;
+  @Mock Dictionary dictionary;
+  @Mock NullabilityHolder nullabilityHolder;
+  @Mock Types.NestedField icebergField;
+
+  VectorHolder vectorHolder;
+
+  @BeforeEach
+  void before() {
+    MockitoAnnotations.initMocks(this);
+    vectorHolder =
+        new VectorHolder(
+            columnDescriptor, vector, false, dictionary, nullabilityHolder, icebergField);
+  }
+
+  @Test
+  void testDescriptor() {
+    ColumnDescriptor result = vectorHolder.descriptor();
+    Assertions.assertSame(this.columnDescriptor, result);
+  }
+
+  @Test
+  void testVector() {
+    FieldVector result = vectorHolder.vector();
```

Comment:
```suggestion
   assertThat(vectorHolder.vector()).isSameAs(vector);
```

please update the other places accordingly

Reply:
I've removed all single-use variables in these unit tests